### ISSUE TITLE
Edit About Us Page

### ISSUE THREAD

**Issue by toliujiayi** [hicodingclub/clubweb#27]

**Comment by andrewnpaul** [comment]
Back in 2018, aspiring programmers and students Jack, Eric and Andrew were requested for help in creating a new website at tutoring center Light of Knowledge Academy. Taking this opportunity, we took on this project as a team to make the new website. After many hours of development through Github, our completed website at lightofknowledgeacademy.com was released by April of 2019. We then had the idea to continue our mission and form a larger team to help the community. And Hi Coding Club was formed to achieve this purpose, As of now, we are still in the beginning phase of our club.

Our mission at Hi Coding Club is to collaborate, learn programming skills, and use our projects to help the local community. Our club is non-profit so we will not make money from our projects. We are willing to tackle challenges, and we'll take interest in any requests for help from our team.

Club president: Jack Liu (toliujiayi@gmail.com)

Outreach: Andrew Paul (andrew.paul@warriorlife.net)

Quality Assurance: Eric Sheng (eric.sheng495@gmail.com)
